2025-02-11 18:47:40,830 - ERROR - Error in application: [Errno 2] No such file or directory: 'D:\\ul8ziz\\GitHub\\thamudic-language-module\\src\\models\\configs\\letter_mapping.json'
2025-02-11 18:47:49,695 - ERROR - Error in application: [Errno 2] No such file or directory: 'D:\\ul8ziz\\GitHub\\thamudic-language-module\\src\\models\\configs\\letter_mapping.json'
2025-02-11 18:50:15,084 - ERROR - Error loading checkpoint: Error(s) in loading state_dict for ThamudicRecognitionModel:
	Missing key(s) in state_dict: "conv1.weight", "bn1.weight", "bn1.bias", "bn1.running_mean", "bn1.running_var", "layer1.0.se.fc.0.weight", "layer1.0.se.fc.2.weight", "layer1.1.se.fc.0.weight", "layer1.1.se.fc.2.weight", "layer2.0.se.fc.0.weight", "layer2.0.se.fc.2.weight", "layer2.1.se.fc.0.weight", "layer2.1.se.fc.2.weight", "layer3.0.se.fc.0.weight", "layer3.0.se.fc.2.weight", "layer3.1.se.fc.0.weight", "layer3.1.se.fc.2.weight", "layer4.0.se.fc.0.weight", "layer4.0.se.fc.2.weight", "layer4.1.se.fc.0.weight", "layer4.1.se.fc.2.weight", "fc.weight", "fc.bias". 
	Unexpected key(s) in state_dict: "conv1.0.weight", "conv1.1.weight", "conv1.1.bias", "conv1.1.running_mean", "conv1.1.running_var", "conv1.1.num_batches_tracked", "layer1.0.se.excitation.0.weight", "layer1.0.se.excitation.2.weight", "layer1.1.se.excitation.0.weight", "layer1.1.se.excitation.2.weight", "layer2.0.se.excitation.0.weight", "layer2.0.se.excitation.2.weight", "layer2.1.se.excitation.0.weight", "layer2.1.se.excitation.2.weight", "layer3.0.se.excitation.0.weight", "layer3.0.se.excitation.2.weight", "layer3.1.se.excitation.0.weight", "layer3.1.se.excitation.2.weight", "layer4.0.se.excitation.0.weight", "layer4.0.se.excitation.2.weight", "layer4.1.se.excitation.0.weight", "layer4.1.se.excitation.2.weight", "fc.1.weight", "fc.1.bias", "fc.4.weight", "fc.4.bias". 
2025-02-11 18:50:15,085 - ERROR - Error loading model: Error(s) in loading state_dict for ThamudicRecognitionModel:
	Missing key(s) in state_dict: "conv1.weight", "bn1.weight", "bn1.bias", "bn1.running_mean", "bn1.running_var", "layer1.0.se.fc.0.weight", "layer1.0.se.fc.2.weight", "layer1.1.se.fc.0.weight", "layer1.1.se.fc.2.weight", "layer2.0.se.fc.0.weight", "layer2.0.se.fc.2.weight", "layer2.1.se.fc.0.weight", "layer2.1.se.fc.2.weight", "layer3.0.se.fc.0.weight", "layer3.0.se.fc.2.weight", "layer3.1.se.fc.0.weight", "layer3.1.se.fc.2.weight", "layer4.0.se.fc.0.weight", "layer4.0.se.fc.2.weight", "layer4.1.se.fc.0.weight", "layer4.1.se.fc.2.weight", "fc.weight", "fc.bias". 
	Unexpected key(s) in state_dict: "conv1.0.weight", "conv1.1.weight", "conv1.1.bias", "conv1.1.running_mean", "conv1.1.running_var", "conv1.1.num_batches_tracked", "layer1.0.se.excitation.0.weight", "layer1.0.se.excitation.2.weight", "layer1.1.se.excitation.0.weight", "layer1.1.se.excitation.2.weight", "layer2.0.se.excitation.0.weight", "layer2.0.se.excitation.2.weight", "layer2.1.se.excitation.0.weight", "layer2.1.se.excitation.2.weight", "layer3.0.se.excitation.0.weight", "layer3.0.se.excitation.2.weight", "layer3.1.se.excitation.0.weight", "layer3.1.se.excitation.2.weight", "layer4.0.se.excitation.0.weight", "layer4.0.se.excitation.2.weight", "layer4.1.se.excitation.0.weight", "layer4.1.se.excitation.2.weight", "fc.1.weight", "fc.1.bias", "fc.4.weight", "fc.4.bias". 
2025-02-11 18:50:15,086 - ERROR - Error in application: Error(s) in loading state_dict for ThamudicRecognitionModel:
	Missing key(s) in state_dict: "conv1.weight", "bn1.weight", "bn1.bias", "bn1.running_mean", "bn1.running_var", "layer1.0.se.fc.0.weight", "layer1.0.se.fc.2.weight", "layer1.1.se.fc.0.weight", "layer1.1.se.fc.2.weight", "layer2.0.se.fc.0.weight", "layer2.0.se.fc.2.weight", "layer2.1.se.fc.0.weight", "layer2.1.se.fc.2.weight", "layer3.0.se.fc.0.weight", "layer3.0.se.fc.2.weight", "layer3.1.se.fc.0.weight", "layer3.1.se.fc.2.weight", "layer4.0.se.fc.0.weight", "layer4.0.se.fc.2.weight", "layer4.1.se.fc.0.weight", "layer4.1.se.fc.2.weight", "fc.weight", "fc.bias". 
	Unexpected key(s) in state_dict: "conv1.0.weight", "conv1.1.weight", "conv1.1.bias", "conv1.1.running_mean", "conv1.1.running_var", "conv1.1.num_batches_tracked", "layer1.0.se.excitation.0.weight", "layer1.0.se.excitation.2.weight", "layer1.1.se.excitation.0.weight", "layer1.1.se.excitation.2.weight", "layer2.0.se.excitation.0.weight", "layer2.0.se.excitation.2.weight", "layer2.1.se.excitation.0.weight", "layer2.1.se.excitation.2.weight", "layer3.0.se.excitation.0.weight", "layer3.0.se.excitation.2.weight", "layer3.1.se.excitation.0.weight", "layer3.1.se.excitation.2.weight", "layer4.0.se.excitation.0.weight", "layer4.0.se.excitation.2.weight", "layer4.1.se.excitation.0.weight", "layer4.1.se.excitation.2.weight", "fc.1.weight", "fc.1.bias", "fc.4.weight", "fc.4.bias". 
2025-02-11 18:50:24,377 - ERROR - Error loading checkpoint: Error(s) in loading state_dict for ThamudicRecognitionModel:
	Missing key(s) in state_dict: "conv1.weight", "bn1.weight", "bn1.bias", "bn1.running_mean", "bn1.running_var", "layer1.0.se.fc.0.weight", "layer1.0.se.fc.2.weight", "layer1.1.se.fc.0.weight", "layer1.1.se.fc.2.weight", "layer2.0.se.fc.0.weight", "layer2.0.se.fc.2.weight", "layer2.1.se.fc.0.weight", "layer2.1.se.fc.2.weight", "layer3.0.se.fc.0.weight", "layer3.0.se.fc.2.weight", "layer3.1.se.fc.0.weight", "layer3.1.se.fc.2.weight", "layer4.0.se.fc.0.weight", "layer4.0.se.fc.2.weight", "layer4.1.se.fc.0.weight", "layer4.1.se.fc.2.weight", "fc.weight", "fc.bias". 
	Unexpected key(s) in state_dict: "conv1.0.weight", "conv1.1.weight", "conv1.1.bias", "conv1.1.running_mean", "conv1.1.running_var", "conv1.1.num_batches_tracked", "layer1.0.se.excitation.0.weight", "layer1.0.se.excitation.2.weight", "layer1.1.se.excitation.0.weight", "layer1.1.se.excitation.2.weight", "layer2.0.se.excitation.0.weight", "layer2.0.se.excitation.2.weight", "layer2.1.se.excitation.0.weight", "layer2.1.se.excitation.2.weight", "layer3.0.se.excitation.0.weight", "layer3.0.se.excitation.2.weight", "layer3.1.se.excitation.0.weight", "layer3.1.se.excitation.2.weight", "layer4.0.se.excitation.0.weight", "layer4.0.se.excitation.2.weight", "layer4.1.se.excitation.0.weight", "layer4.1.se.excitation.2.weight", "fc.1.weight", "fc.1.bias", "fc.4.weight", "fc.4.bias". 
2025-02-11 18:50:24,378 - ERROR - Error loading model: Error(s) in loading state_dict for ThamudicRecognitionModel:
	Missing key(s) in state_dict: "conv1.weight", "bn1.weight", "bn1.bias", "bn1.running_mean", "bn1.running_var", "layer1.0.se.fc.0.weight", "layer1.0.se.fc.2.weight", "layer1.1.se.fc.0.weight", "layer1.1.se.fc.2.weight", "layer2.0.se.fc.0.weight", "layer2.0.se.fc.2.weight", "layer2.1.se.fc.0.weight", "layer2.1.se.fc.2.weight", "layer3.0.se.fc.0.weight", "layer3.0.se.fc.2.weight", "layer3.1.se.fc.0.weight", "layer3.1.se.fc.2.weight", "layer4.0.se.fc.0.weight", "layer4.0.se.fc.2.weight", "layer4.1.se.fc.0.weight", "layer4.1.se.fc.2.weight", "fc.weight", "fc.bias". 
	Unexpected key(s) in state_dict: "conv1.0.weight", "conv1.1.weight", "conv1.1.bias", "conv1.1.running_mean", "conv1.1.running_var", "conv1.1.num_batches_tracked", "layer1.0.se.excitation.0.weight", "layer1.0.se.excitation.2.weight", "layer1.1.se.excitation.0.weight", "layer1.1.se.excitation.2.weight", "layer2.0.se.excitation.0.weight", "layer2.0.se.excitation.2.weight", "layer2.1.se.excitation.0.weight", "layer2.1.se.excitation.2.weight", "layer3.0.se.excitation.0.weight", "layer3.0.se.excitation.2.weight", "layer3.1.se.excitation.0.weight", "layer3.1.se.excitation.2.weight", "layer4.0.se.excitation.0.weight", "layer4.0.se.excitation.2.weight", "layer4.1.se.excitation.0.weight", "layer4.1.se.excitation.2.weight", "fc.1.weight", "fc.1.bias", "fc.4.weight", "fc.4.bias". 
2025-02-11 18:50:24,378 - ERROR - Error in application: Error(s) in loading state_dict for ThamudicRecognitionModel:
	Missing key(s) in state_dict: "conv1.weight", "bn1.weight", "bn1.bias", "bn1.running_mean", "bn1.running_var", "layer1.0.se.fc.0.weight", "layer1.0.se.fc.2.weight", "layer1.1.se.fc.0.weight", "layer1.1.se.fc.2.weight", "layer2.0.se.fc.0.weight", "layer2.0.se.fc.2.weight", "layer2.1.se.fc.0.weight", "layer2.1.se.fc.2.weight", "layer3.0.se.fc.0.weight", "layer3.0.se.fc.2.weight", "layer3.1.se.fc.0.weight", "layer3.1.se.fc.2.weight", "layer4.0.se.fc.0.weight", "layer4.0.se.fc.2.weight", "layer4.1.se.fc.0.weight", "layer4.1.se.fc.2.weight", "fc.weight", "fc.bias". 
	Unexpected key(s) in state_dict: "conv1.0.weight", "conv1.1.weight", "conv1.1.bias", "conv1.1.running_mean", "conv1.1.running_var", "conv1.1.num_batches_tracked", "layer1.0.se.excitation.0.weight", "layer1.0.se.excitation.2.weight", "layer1.1.se.excitation.0.weight", "layer1.1.se.excitation.2.weight", "layer2.0.se.excitation.0.weight", "layer2.0.se.excitation.2.weight", "layer2.1.se.excitation.0.weight", "layer2.1.se.excitation.2.weight", "layer3.0.se.excitation.0.weight", "layer3.0.se.excitation.2.weight", "layer3.1.se.excitation.0.weight", "layer3.1.se.excitation.2.weight", "layer4.0.se.excitation.0.weight", "layer4.0.se.excitation.2.weight", "layer4.1.se.excitation.0.weight", "layer4.1.se.excitation.2.weight", "fc.1.weight", "fc.1.bias", "fc.4.weight", "fc.4.bias". 
2025-02-11 18:51:19,571 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-11 18:51:19,572 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-11 18:51:31,575 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-11 18:51:31,575 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-11 18:51:41,868 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-11 18:51:41,870 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-11 18:59:36,583 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-11 18:59:36,587 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-11 18:59:40,250 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-11 18:59:40,250 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-11 18:59:40,418 - ERROR - Error making prediction: Input type (torch.cuda.FloatTensor) and weight type (torch.FloatTensor) should be the same
2025-02-11 18:59:40,420 - ERROR - Error in application: Input type (torch.cuda.FloatTensor) and weight type (torch.FloatTensor) should be the same
2025-02-11 19:04:15,124 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-11 19:04:15,124 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-11 19:04:18,533 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-11 19:04:18,533 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-11 19:04:18,742 - ERROR - Error making prediction: Input type (torch.cuda.FloatTensor) and weight type (torch.FloatTensor) should be the same
2025-02-11 19:04:18,743 - ERROR - Error in application: Input type (torch.cuda.FloatTensor) and weight type (torch.FloatTensor) should be the same
2025-02-11 19:08:46,409 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-11 19:08:46,409 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-11 19:08:52,534 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-11 19:08:52,535 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-13 17:51:43,401 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-13 17:51:43,404 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-13 17:51:48,339 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-13 17:51:48,340 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-13 18:07:14,845 - ERROR - Error in application: [Errno 2] No such file or directory: 'D:\\ul8ziz\\GitHub\\thamudic-language-module\\src\\models\\configs\\letter_mapping.json'
2025-02-13 18:11:17,575 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-13 18:11:17,577 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-13 18:11:22,027 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-13 18:11:22,030 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-13 18:13:36,344 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-13 18:13:36,348 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-13 18:13:46,635 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-13 18:13:46,637 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-13 18:16:26,208 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-13 18:16:26,210 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-13 18:16:30,808 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-13 18:16:30,810 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 16:51:17,174 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 16:51:17,178 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 16:51:20,255 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 16:51:20,256 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 16:54:57,982 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 16:54:57,983 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 16:55:01,538 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 16:55:01,539 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 16:55:01,620 - ERROR - Error in application: 'ThamudicApp' object has no attribute 'predict'
2025-02-14 16:57:31,419 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 16:57:31,421 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 16:57:34,774 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 16:57:34,775 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 16:57:34,903 - ERROR - Error making prediction: 'box_left'
2025-02-14 16:57:34,904 - ERROR - Error in application: 'box_left'
2025-02-14 16:58:34,196 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 16:58:34,198 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 16:58:37,935 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 16:58:37,938 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:03:59,037 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:03:59,038 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:04:02,369 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:04:02,371 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:04:03,339 - ERROR - Error in application: name 'confidences' is not defined
2025-02-14 17:05:54,218 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:05:54,221 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:05:57,185 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:05:57,188 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:05:58,089 - ERROR - Error in application: name 'confidences' is not defined
2025-02-14 17:06:09,050 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:06:09,051 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:06:12,131 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:06:12,133 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:06:13,001 - ERROR - Error in application: name 'confidences' is not defined
2025-02-14 17:09:12,761 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:09:12,762 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:09:15,643 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:09:15,645 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:09:16,541 - ERROR - Error in application: name 'confidences' is not defined
2025-02-14 17:12:31,158 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:12:31,159 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:12:34,182 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:12:34,184 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:12:35,073 - ERROR - Error in application: name 'confidences' is not defined
2025-02-14 17:13:40,650 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:13:40,651 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:13:43,823 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:13:43,824 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:13:44,715 - ERROR - Error in application: name 'confidences' is not defined
2025-02-14 17:15:39,617 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:15:39,620 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:15:42,611 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:15:42,612 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:15:43,498 - ERROR - Error in application: name 'confidences' is not defined
2025-02-14 17:16:47,321 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:16:47,323 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:16:50,188 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:16:50,190 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:16:51,063 - ERROR - Error in application: name 'confidences' is not defined
2025-02-14 17:17:11,970 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:17:11,971 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:17:14,577 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:17:14,579 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-14 17:17:15,445 - ERROR - Error in application: name 'confidences' is not defined
2025-02-15 17:07:38,908 - ERROR - Error loading checkpoint: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:07:38,911 - ERROR - Error loading model: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:07:38,912 - ERROR - Error in application: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:12:03,888 - ERROR - Error loading checkpoint: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:12:03,889 - ERROR - Error loading model: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:12:03,891 - ERROR - Error in application: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:14:05,045 - ERROR - Error loading checkpoint: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:14:05,046 - ERROR - Error loading model: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:14:05,048 - ERROR - Error in application: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:16:13,146 - ERROR - Error loading checkpoint: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:16:13,147 - ERROR - Error loading model: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:16:13,147 - ERROR - Error in application: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:19:36,524 - ERROR - Error loading checkpoint: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:19:36,530 - ERROR - Error loading model: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:19:36,537 - ERROR - Error in application: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:20:28,058 - ERROR - Error loading checkpoint: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:20:28,058 - ERROR - Error loading model: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:20:28,058 - ERROR - Error in application: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:20:50,639 - ERROR - Error loading checkpoint: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:20:50,682 - ERROR - Error loading model: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:20:50,730 - ERROR - Error in application: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:21:06,283 - ERROR - Error loading checkpoint: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:21:06,293 - ERROR - Error loading model: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:21:06,318 - ERROR - Error in application: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:44:14,922 - ERROR - Error loading checkpoint: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:44:14,932 - ERROR - Error loading model: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:44:14,939 - ERROR - Error loading checkpoint: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:44:14,943 - ERROR - Error in application: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:44:14,944 - ERROR - Error loading model: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:44:14,949 - ERROR - Error in application: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:44:15,729 - ERROR - Error loading checkpoint: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:44:15,743 - ERROR - Error loading model: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:44:15,748 - ERROR - Error in application: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:44:15,776 - ERROR - Error loading checkpoint: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:44:15,780 - ERROR - Error loading model: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:44:16,102 - ERROR - Error in application: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:45:04,790 - ERROR - Error loading checkpoint: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:45:04,812 - ERROR - Error loading model: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:45:04,838 - ERROR - Error in application: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:45:29,914 - ERROR - Error loading checkpoint: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:45:29,919 - ERROR - Error loading model: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-15 17:45:29,923 - ERROR - Error in application: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-16 13:20:54,651 - ERROR - Error loading checkpoint: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-16 13:20:54,653 - ERROR - Error loading model: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-16 13:20:54,655 - ERROR - Error in application: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-16 13:28:25,099 - ERROR - Error loading checkpoint: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-16 13:28:25,100 - ERROR - Error loading model: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-16 13:28:25,101 - ERROR - Error in application: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-16 13:28:40,475 - ERROR - Error loading checkpoint: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-16 13:28:40,476 - ERROR - Error loading model: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-16 13:28:40,477 - ERROR - Error in application: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torch.optim.adamw.AdamW was not an allowed global by default. Please use `torch.serialization.add_safe_globals([AdamW])` or the `torch.serialization.safe_globals([AdamW])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-02-16 13:29:43,793 - ERROR - Error loading checkpoint: 'str' object has no attribute '__module__'
2025-02-16 13:29:43,793 - ERROR - Error loading model: 'str' object has no attribute '__module__'
2025-02-16 13:29:43,795 - ERROR - Error in application: 'str' object has no attribute '__module__'
2025-02-16 13:33:32,096 - ERROR - Error loading checkpoint: Error(s) in loading state_dict for ThamudicRecognitionModel:
	size mismatch for fc.4.weight: copying a param with shape torch.Size([27, 256]) from checkpoint, the shape in current model is torch.Size([28, 256]).
	size mismatch for fc.4.bias: copying a param with shape torch.Size([27]) from checkpoint, the shape in current model is torch.Size([28]).
2025-02-16 13:33:32,096 - ERROR - Error loading model: Error(s) in loading state_dict for ThamudicRecognitionModel:
	size mismatch for fc.4.weight: copying a param with shape torch.Size([27, 256]) from checkpoint, the shape in current model is torch.Size([28, 256]).
	size mismatch for fc.4.bias: copying a param with shape torch.Size([27]) from checkpoint, the shape in current model is torch.Size([28]).
2025-02-16 13:33:32,098 - ERROR - Error in application: Error(s) in loading state_dict for ThamudicRecognitionModel:
	size mismatch for fc.4.weight: copying a param with shape torch.Size([27, 256]) from checkpoint, the shape in current model is torch.Size([28, 256]).
	size mismatch for fc.4.bias: copying a param with shape torch.Size([27]) from checkpoint, the shape in current model is torch.Size([28]).
2025-02-16 13:34:18,873 - WARNING - Mismatch in fc.4.weight: checkpoint shape torch.Size([27, 256]), current model shape torch.Size([28, 256])
2025-02-16 13:34:18,877 - WARNING - Mismatch in fc.4.bias: checkpoint shape torch.Size([27]), current model shape torch.Size([28])
2025-02-16 13:34:18,895 - INFO - Checkpoint loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-16 13:34:18,905 - INFO - Model loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-16 13:34:26,333 - WARNING - Mismatch in fc.4.weight: checkpoint shape torch.Size([27, 256]), current model shape torch.Size([28, 256])
2025-02-16 13:34:26,333 - WARNING - Mismatch in fc.4.bias: checkpoint shape torch.Size([27]), current model shape torch.Size([28])
2025-02-16 13:34:26,355 - INFO - Checkpoint loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-16 13:34:26,359 - INFO - Model loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-16 13:37:46,180 - INFO - Checkpoint loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-16 13:37:46,185 - INFO - Model loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-16 13:37:50,456 - INFO - Checkpoint loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-16 13:37:50,460 - INFO - Model loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-16 14:01:33,104 - INFO - Checkpoint loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-16 14:01:33,121 - INFO - Model loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-16 14:01:33,616 - INFO - Checkpoint loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-16 14:01:33,621 - INFO - Model loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-16 14:01:59,868 - INFO - Checkpoint loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-16 14:01:59,874 - INFO - Model loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-16 14:02:07,175 - INFO - Checkpoint loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-16 14:02:07,176 - INFO - Model loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-16 14:06:00,886 - INFO - Checkpoint loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-16 14:06:00,894 - INFO - Model loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-16 14:06:08,711 - INFO - Checkpoint loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-16 14:06:08,715 - INFO - Model loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-16 14:06:47,349 - INFO - Checkpoint loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-16 14:06:47,375 - INFO - Model loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-16 14:06:47,869 - INFO - Checkpoint loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-16 14:06:47,873 - INFO - Model loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-16 14:08:03,713 - INFO - Checkpoint loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-16 14:08:03,718 - INFO - Model loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-16 14:08:09,880 - INFO - Checkpoint loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-16 14:08:09,886 - INFO - Model loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-16 14:11:34,746 - INFO - Checkpoint loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-16 14:11:34,751 - INFO - Model loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-16 14:11:41,088 - INFO - Checkpoint loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-16 14:11:41,092 - INFO - Model loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-22 23:32:11,519 - WARNING - Mismatch in fc.4.weight: checkpoint shape torch.Size([27, 256]), current model shape torch.Size([28, 256])
2025-02-22 23:32:11,523 - WARNING - Mismatch in fc.4.bias: checkpoint shape torch.Size([27]), current model shape torch.Size([28])
2025-02-22 23:32:11,551 - INFO - Checkpoint loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-22 23:32:11,556 - INFO - Model loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-22 23:32:20,946 - WARNING - Mismatch in fc.4.weight: checkpoint shape torch.Size([27, 256]), current model shape torch.Size([28, 256])
2025-02-22 23:32:20,946 - WARNING - Mismatch in fc.4.bias: checkpoint shape torch.Size([27]), current model shape torch.Size([28])
2025-02-22 23:32:20,980 - INFO - Checkpoint loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-22 23:32:20,986 - INFO - Model loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-23 00:11:58,246 - WARNING - Mismatch in fc.4.weight: checkpoint shape torch.Size([27, 256]), current model shape torch.Size([28, 256])
2025-02-23 00:11:58,246 - WARNING - Mismatch in fc.4.bias: checkpoint shape torch.Size([27]), current model shape torch.Size([28])
2025-02-23 00:11:58,276 - INFO - Checkpoint loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-23 00:11:58,282 - INFO - Model loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-23 00:12:02,242 - WARNING - Mismatch in fc.4.weight: checkpoint shape torch.Size([27, 256]), current model shape torch.Size([28, 256])
2025-02-23 00:12:02,243 - WARNING - Mismatch in fc.4.bias: checkpoint shape torch.Size([27]), current model shape torch.Size([28])
2025-02-23 00:12:02,262 - INFO - Checkpoint loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-23 00:12:02,266 - INFO - Model loaded successfully from D:\GitHub\Rizq\thamudic-language-module\models\checkpoints\best_model.pt
2025-02-25 14:41:12,504 - ERROR - Error loading checkpoint: [Errno 2] No such file or directory: 'D:\\ul8ziz\\GitHub\\thamudic-language-module\\models\\checkpoints\\best_model.pt'
2025-02-25 14:41:12,506 - ERROR - Error loading model: [Errno 2] No such file or directory: 'D:\\ul8ziz\\GitHub\\thamudic-language-module\\models\\checkpoints\\best_model.pt'
2025-02-25 14:41:12,506 - ERROR - Error in application: [Errno 2] No such file or directory: 'D:\\ul8ziz\\GitHub\\thamudic-language-module\\models\\checkpoints\\best_model.pt'
2025-02-25 14:42:36,569 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pth
2025-02-25 14:42:36,573 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pth
2025-02-25 14:42:48,083 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pth
2025-02-25 14:42:48,085 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pth
2025-02-25 14:44:36,151 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pth
2025-02-25 14:44:36,155 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pth
2025-02-25 14:46:10,140 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pth
2025-02-25 14:46:10,143 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pth
2025-02-25 14:46:16,302 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pth
2025-02-25 14:46:16,302 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pth
2025-02-25 14:46:16,330 - ERROR - Error making prediction: OpenCV(4.7.0) d:\a\opencv-python\opencv-python\opencv\modules\imgproc\src\color.simd_helpers.hpp:92: error: (-2:Unspecified error) in function '__cdecl cv::impl::`anonymous-namespace'::CvtHelper<struct cv::impl::`anonymous namespace'::Set<3,4,-1>,struct cv::impl::A0x981fb336::Set<1,-1,-1>,struct cv::impl::A0x981fb336::Set<0,2,5>,2>::CvtHelper(const class cv::_InputArray &,const class cv::_OutputArray &,int)'
> Invalid number of channels in input image:
>     'VScn::contains(scn)'
> where
>     'scn' is 1

2025-02-25 14:46:16,333 - ERROR - Error in application: OpenCV(4.7.0) d:\a\opencv-python\opencv-python\opencv\modules\imgproc\src\color.simd_helpers.hpp:92: error: (-2:Unspecified error) in function '__cdecl cv::impl::`anonymous-namespace'::CvtHelper<struct cv::impl::`anonymous namespace'::Set<3,4,-1>,struct cv::impl::A0x981fb336::Set<1,-1,-1>,struct cv::impl::A0x981fb336::Set<0,2,5>,2>::CvtHelper(const class cv::_InputArray &,const class cv::_OutputArray &,int)'
> Invalid number of channels in input image:
>     'VScn::contains(scn)'
> where
>     'scn' is 1

2025-02-25 14:46:35,320 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pth
2025-02-25 14:46:35,321 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pth
2025-02-25 14:51:28,574 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pth
2025-02-25 14:51:28,579 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pth
2025-02-25 14:51:47,458 - INFO - Checkpoint loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pth
2025-02-25 14:51:47,460 - INFO - Model loaded successfully from D:\ul8ziz\GitHub\thamudic-language-module\models\checkpoints\best_model.pth
